src/main/scala/com/gu/restaurant_review_parser/ParsedRestaurantReview.scala

```diff
@@ -0,0 +1,49 @@
+package com.gu.restaurant_review_parser
+
+import java.time.OffsetDateTime
+
+case class WebAddress(value: String) extends AnyVal
+case class Address(value: String) extends AnyVal
+case class ArticleBody(value: String) extends AnyVal
+case class RestaurantName(value: String) extends AnyVal
+case class ApproximateLocation(value: String) extends AnyVal
+case class WebTitle(value: String) extends AnyVal
+
+sealed trait RatingBreakdownType {
+  val rating: String
+}
+case class FoodRating(rating: String) extends RatingBreakdownType
+case class AtmosphereRating(rating: String) extends RatingBreakdownType
+case class ValueForMoneyRating(rating: String) extends RatingBreakdownType
+
+case class RatingBreakdown(food: FoodRating, atmosphere: AtmosphereRating, valueForMoney: ValueForMoneyRating)
+
+case class ParsedRestaurantReview (
+  restaurantName: RestaurantName,
+  approximateLocation: ApproximateLocation,
+  reviewer: String,
+  publicationDate: OffsetDateTime,
+  ratingBreakdown: Option[RatingBreakdown],
+  address: Option[Address], // TODO - structure accordingly
+  webAddress: Option[WebAddress]
```

Comment:
Do we want to include phone number and opening hours? Or will these go under address?